### PR TITLE
COMPOSER: Support MacBinary format for Macintosh V1 games

### PR DESCRIPTION
**Pull request by BLooperZ**
This allows reading book configuration file when using the recommended method of extracting files from Macintosh CDs
https://docs.scummvm.org/en/v2.6.1/use_scummvm/mac_game_files.html#python-dumper

fixes some detection entries (previously added with different extraction method)

### CODE REVIEW

File: engines/composer/detection_tables.h [hunk 1]
```diff
@@ -148,7 +148,7 @@ static const ComposerGameDescription gameDescriptions[] = {
 			"imoking",
 			0,
 			{
-				{"book.mac", GAME_CONFIGFILE, "4896a22874bb660f5ba26a0af111f9c0", 1868},
+				{"book.mac", GAME_CONFIGFILE, "916757dbb058581cff857b055ec97642", 2560},
```
Comment (phcoder) [inline engines/composer/detection_tables.h:151]: Please use ADGF_MACRESFORK to make hashes independent from extraction method

Comment (BLooperZ) [reply engines/composer/detection_tables.h]: Awesome, thank you

File: engines/composer/detection_tables.h [hunk 2]
```diff
@@ -352,7 +352,7 @@ static const ComposerGameDescription gameDescriptions[] = {
 	{
 		{
 			"magictales",
-			"Magic Tales Demo: Baby Yaga, Samurai, Imo",
+			"Magic Tales Demo: Baba Yaga, Samurai, Imo",
```
[the remark this answers is not in the document]
Comment (BLooperZ) [reply engines/composer/detection_tables.h:355]: done

**Review by phcoder — APPROVED** [review]

**Comment by sev-** [comment]
Thank you!

**Pull request merged**